contrib/raftexample/raft.go

```diff
@@ -460,7 +460,8 @@ func (rc *raftNode) serveRaft() {
 		log.Fatalf("raftexample: Failed parsing URL (%v)", err)
 	}
 
-	ln, err := newStoppableListener(url.Host, rc.httpstopc)
+	addr := fmt.Sprintf(":%s", url.Port())
```

Comment:
@johncming aren't they two different use cases i.e. the current one which uses host (host with or without port) and :port, and we should handle both? Thanks!

Reply:
> @johncming aren't they two different use cases i.e. the current one which uses host (host with or without port) and :port, and we should handle both? Thanks!

which two case,  is this?
1. host:port
2. host only

Reply:
Yes with host (host:port, host only) and :port